jmix-jmxconsole/jmxconsole/src/main/java/io/jmix/jmxconsole/JmxConsoleConfiguration.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2022 Haulmont.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.jmix.jmxconsole;
+
+import io.jmix.core.annotation.JmixModule;
+import io.jmix.core.impl.scanning.AnnotationScanMetadataReaderFactory;
+import io.jmix.flowui.FlowuiConfiguration;
+import io.jmix.flowui.sys.ActionsConfiguration;
+import io.jmix.flowui.sys.ViewControllersConfiguration;
+import org.springframework.boot.context.properties.ConfigurationPropertiesScan;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+
+import java.util.Collections;
+
+@Configuration
+@ComponentScan
+@ConfigurationPropertiesScan
+@JmixModule(dependsOn = {FlowuiConfiguration.class})
+@PropertySource(name = "io.jmix.jmxconsole", value = "classpath:/io/jmix/jmxconsole/module.properties")
+public class JmxConsoleConfiguration {
+    @Bean("jmxcon_JmxconViewControllers")
+    public ViewControllersConfiguration screens(final ApplicationContext applicationContext,
+                                                final AnnotationScanMetadataReaderFactory metadataReaderFactory) {
+        final ViewControllersConfiguration viewControllers
+                = new ViewControllersConfiguration(applicationContext, metadataReaderFactory);
+        viewControllers.setBasePackages(Collections.singletonList("io.jmix.jmxconsole"));
+        return viewControllers;
+    }
+
+    @Bean("jmxcon_JmxconActions")
+    public ActionsConfiguration actions(final ApplicationContext applicationContext,
+                                        final AnnotationScanMetadataReaderFactory metadataReaderFactory) {
+        final ActionsConfiguration actions
+                = new ActionsConfiguration(applicationContext, metadataReaderFactory);
+        actions.setBasePackages(Collections.singletonList("io.jmix.jmxconsole"));
```

Comment:
1. do you have actions?
2. Actions should be located in the `xxx.action` package, no need to make platform scan all the packages in the module to find actions.